release_description.md

```diff
@@ -0,0 +1,30 @@
+The Amazon QLDB team is pleased to announce the release of v1.2.0 of the QLDB .NET driver.
+
+## Enhancements
+* Add the asynchronous version of the driver.
```

Comment:
nit - be consistent on the tense. Below says "Improved", so use "Added" here.

Reply:
Done.